test/buildah-bud/apply-podman-deltas

```diff
@@ -188,6 +201,15 @@ skip_if_remote "local keyfile not sent to podman-remote" \
 skip_if_remote "--stdin option will not be implemented in podman-remote" \
                "bud test no --stdin"
 
+# https://github.com/containers/buildah/pull/3823
+# If this is possible with podman-remote, it'll take way more Go skills
+# to implement than what Ed can do.
+skip_if_remote "--output option not implemented in podman-remote" \
+               "build with custom build output and output rootfs to directory" \
+               "build with custom build output and output rootfs to tar" \
+               "build with custom build output and output rootfs to tar by pipe" \
+               "build with custom build output must fail for bad input"
+
```

Comment:
If podman-remote does not support this, then the code should error when the option is used.
Second it has to be documented in the --output option that this is not support via remote.

Reply:
@Luap99 Yes I'll open a followup after this vendor to address this.